### ISSUE TITLE
suggestion of adding a new term (Hemophilia)

### ISSUE THREAD

**Issue by yuhkalhic** [obophenotype/human-phenotype-ontology#11439]
**Preferred term label:**
Hemophilia

**Synonyms**

**Definition (free text, please give PubMed ID)**

Hemophilia is a hereditary bleeding disorder caused by the deficiency or dysfunction of coagulation factor VIII (hemophilia A) or factor IX (hemophilia B), resulting in prolonged and excessive bleeding either spontaneously or secondary to trauma.

PMID: 32744769 (Srivastava et al., 2020, WFH Guidelines for the Management of Hemophilia, 3rd edition)
PMID: 22776238 (Srivastava et al., 2013, Guidelines for the management of hemophilia, 2nd edition)

**Parent term (use hpo.jax.org/app)**

Phenotypic abnormality → Abnormality of blood and blood-forming tissues → Abnormality of coagulation → Abnormal bleeding → Hemophilia → Hemophilia A/B

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

OMIM #306700 — Hemophilia A (F8 gene, Xq28)
OMIM #306900 — Hemophilia B (F9 gene, Xq27.1)
OMIM #134500 — Factor VIII deficiency, autosomal
Orphanet: ORPHA:98878 (Hemophilia A), ORPHA:98879 (Hemophilia B)
ORPHA:169802 / 169805 / 169808 (Severe / Moderate / Mild Hemophilia A)
ORPHA:169793 / 169796 / 169799 (Severe / Moderate / Mild Hemophilia B)

**Your nano-attribution (ORCID)**

**Comment by yuhkalhic** [comment]
@pnrobinson cc

**Comment by pnrobinson** [comment]
@yuhkalhic 
Thanks for the issue. 
Hemophilia is a disease and we have disease models for it (e.g. https://hpo.jax.org/browse/disease/OMIM:306700).
It is important to keep diseases and phenotypic features strictly separated for the HPO algorithms to work, therefore this "Hemophilia" is not a proper HPO term; see this https://obophenotype.github.io/human-phenotype-ontology/clinicians/what-is-phenotype/
